UnitsNet.NanoFramework/GeneratedCode/Temperature/UnitsNet.NanoFramework.Temperature.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>UnitsNet.nanoFramework.Temperature</id>
+    <version>4.89.0</version>
+    <title>Units.NET - Temperature</title>
+    <authors>Andreas Gullberg Larsen</authors>
+    <owners>Andreas Gullberg Larsen</owners>
+    <license type="expression">MIT</license>
```

Comment:
I propose you put your own names or github usernames as `authors`, if you like. Not my name.

For `owners`, I believe we can put `UnitsNet`, but it might not matter as it seems maybe the nuget.org API key decides the owner field when published.

Reply:
Added the contributors mention used in .NET project

Reply:
Regarding owners: this was taken from the other nuspecs that you have there. 
I've changed that to UnitsNet as that's what shows on NuGet.

I don't think I can add nanoFramework because our packages are required to be signed.

Reply:
Ok, it's fine I think.